openid-connect-server/src/main/java/org/mitre/openid/connect/service/impl/MITREidDataService_1_0.java

```diff
@@ -835,7 +835,7 @@ private void fixObjectReferences() {
             }
             Long newGrantId = grantOldToNewIdMap.get(oldGrantId);
             ApprovedSite site = approvedSiteRepository.getById(newGrantId);
-            site.setApprovedAccessTokens(tokens);
+//            site.setApprovedAccessTokens(tokens);
```

Comment:
This looks like it would drop the approved site -> token binding during import from a 1.0 data export. There needs to be code to save this binding and put it in the right place.